src/components/Glossary/GlossaryTooltip/index.tsx

```diff
@@ -13,7 +13,15 @@ const GlossaryTooltip = ({ children, termKey }: GlossaryTooltipProps) => {
   const isLargeScreen = useBreakpointValue({ base: false, lg: true })
 
   return isLargeScreen ? (
-    <Tooltip content={<GlossaryDefinition term={termKey} size="sm" />}>
+    <Tooltip
```

Comment:
Probably for the backlog (solved by Chakra?), but collision handling isn't great. Don't think it's blocking as it's pretty standard behaviour for this type of feature.

<img width="1466" alt="Screenshot 2024-03-05 at 08 48 23" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/772ae9ea-ca02-4c13-beed-a42bd9299230">

Reply:
we have discussed this one with pablo already, chakra tooltip should be fixing htis

Reply:
Yes, this is going to be fixed in the tooltip chakra migration #10974